src/userInterface/panels/JsonConfigurator/JsonConfigurator.tsx

```diff
@@ -6,6 +6,13 @@ import { JsonConfigCommandCenter } from "../../../core/JsonConfigCommandCenter";
 import { CommandPanel } from "../CommandPanel/CommandPanel";
 import { SideNavPanel } from '../SideNavPanel/SideNavPanel';
 import { EditorPanel } from '../EditorPanel/EditorPanel';
+import { message } from 'antd';
```

Comment:
use specific import whenere possible to reduce library weight like "antd/es/message"